test/apps/iodemo/io_demo.cc

```diff
@@ -764,11 +815,35 @@ class DemoClient : public P2pDemoCommon {
             if (opts().window_size == 1) {
                 std::cout << ", average latency: " << latency_usec << " usec";
             }
+            if (opts().print_conn_status) {
+                long min = _conn[0].num_completed;
+                long max = _conn[0].num_completed;
```

Comment:
- instead of num_completed, count read_bytes and write_bytes for every connection
- do not wait for responses before printing , instead print by read_bytes and write_bytes
- after the print, set read_bytes and write_bytes to 0, so every print will report only for the previous interval and not since the beginning of the test